keyboards/eyeohdesigns/Readme.md

```diff
@@ -0,0 +1,13 @@
+eye oh designs
```

Comment:
Could you rename the file, so that it's all lower case? 

This prevents issues on some OS's with capitalization.

Reply:
Fixed.